### ISSUE TITLE
RuntimeWarning in estnoisems function

### ISSUE THREAD

**Issue by newrlan** [eesungkim/Voice_Activity_Detector#1]
Hi, I found a mistake when run VAD on a single. If the pSpectrum value has only zeroes on first string, the result of `estnoisem` function return table with `np.nan`.  I added strings

```
begin = 0
while begin < nFrames and np.all(pSpectrum[begin,:] == 0.0):
        begin += 1
p=pSpectrum[begin,:]  
```
to redefinition `p` valuation and run `for` cycle in `range(begin, nFrames)`.

That improve a crashing of code, but hold that mistake
```
Voice_Activity_Detector/utils/estnoise_ms.py:108: RuntimeWarning: divide by zero encountered in double_scalars
  acb=(1+(sum(p) / sum(pSpectrum_t)-1)**2)**(-1)    # alpha_c-bar(t)  (9)
```

**Comment by nycsv** [comment]
Yes, I also think there is a problem on that part. Initial 'p' should be not zero or that big value comparing other frames.

Thank you for your feedback.

**Comment by krupalraj** [comment]
@proposition does that fix the crashing of code ??
iam having the same issue

**Comment by newrlan** [comment]
@krupalraj yes, I did this, but I've not saved the code version.

The problem appears when your audio file have zero parts. The simplest way to avoid the problem is to cut off the zero regions.

**Comment by krupalraj** [comment]
sure,Thanks @proposition
